web/versions.go

```diff
@@ -259,6 +299,13 @@ func getVersion(c echo.Context) error {
 		return err
 	}
 
+	oDoc, e := override(c, doc)
```

Comment:
Can't we make `override` returns the original `doc` when there is no overriding? It will avoid the 3 lines below:

```go
	if oDoc != nil {
		doc = oDoc
	}
```